zipline/finance/slippage.py

```diff
@@ -314,10 +314,13 @@ def __init__(self, spread=0.0):
         self.spread = spread
 
     def process_order(self, event, order, dt):
```

Comment:
It looks like this no longer needs to be passed the event, which raises the question: **anything** should ever get passed an `Event` object anymore?  It seems like the answer should be "No"?